src/components/BulkEnrollmentPage/stepper/AddLearnersStep.jsx

```diff
@@ -43,14 +41,68 @@ const selectColumn = {
   disableSortBy: true,
 };
 
+const tableColumns = [
+  selectColumn,
+  {
+    Header: TABLE_HEADERS.email,
+    accessor: 'userEmail',
+    Filter: TextFilter,
+  },
+];
+
+const INITIAL_PAGE_INDEX = 0;
+const PAGE_SIZE = 5;
+
+const useIsMounted = () => {
+  const componentIsMounted = useRef(true);
+  useEffect(() => () => { componentIsMounted.current = false; }, []);
+  return componentIsMounted;
+};
+
 const AddLearnersStep = ({
   subscriptionUUID,
   enterpriseSlug,
 }) => {
-  const [errors, setErrors] = useState([]);
+  const [errors, setErrors] = useState('');
+  const [loading, setLoading] = useState(true);
+  const [data, setData] = useState({ results: [], count: 0, numPages: 1 });
   const { emails: [selectedEmails] } = useContext(BulkEnrollContext);
+  const { results, count, numPages } = data;
```

Comment:
Surprised there isn't a way to just combo set this at the time you set data. Something like `const [data{results, count, numPages}, setData] = useState...`

Reply:
Oh, yeah, it can definitely be done this way. I probably split it up like this for debugging purposes.